.prettierrc.json

```diff
@@ -1,11 +1,11 @@
 {
-  "printWidth": 240,
+  "printWidth": 80,
   "tabWidth": 2,
   "useTabs": false,
   "semi": true,
   "singleQuote": true,
-  "trailingComma": "none",
-  "bracketSpacing": false,
+  "trailingComma": "all",
+  "bracketSpacing": true,
```

Comment:
Could we look at prettier format change in a dedicated PR as follow-up? Current rules are also used in actions repo so this might be smth we want to consider across repos.

Reply:
Yup happy to.

Reply:
Alrighty, undid the prettier changes. Will followup maybe a discussion on improving code stuff there.